File: .net & mono/Weed3Demo/test/model/UserModel.cs
using Noear.Weed;
using System;

namespace Weed3Demo.test.model {
    public class UserModel : IBinder {
        public long UserID;
        public String Nickname;
        public int Sex;
        public String Icon;
        public String City;

        public void bind(GetHandlerEx s) {
            UserID   = s("UserID").value(0L);
            Nickname = s("Nickname").value("");
            Sex      = s("Sex").value(0);
            Icon     = s("Icon").value("");
            City     = s("City").value("");
        }

        public IBinder clone() {
            return new UserModel();
        }
    }
}

File: _test/src/test/java/weed3mongo/model/UserModel.java
package weed3mongo.model;

import java.io.Serializable;

/**
 * @author noear 2021/2/1 created
 */

public class UserModel implements Serializable {
    public long userId;
    public int type;
    public String name;
    public String nickName;

    @Override
    public String toString() {
        return "UserModel{" +
                "userId=" + userId +
                ", type=" + type +
                ", name='" + name + '\'' +
                ", nickName='" + nickName + '\'' +
                '}';
    }
}
